decentraland/unity-renderer
# Chore: various avatar preview improvements

## What does this PR change?

Fix #1948 
Adds mouse wheel control on character preview and different ui improvements

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/avatar-improvements
2. Go into backpack
3. Verify empty backpack messages
4. Try to zoom in and out on character preview

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
